mne_bids/meg_bids.py

```diff
@@ -337,7 +347,8 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
 
     # save stuff
     if kind == 'meg':
-        _scans_tsv(raw, raw_file_bids, scans_fname, verbose)
+        _scans_tsv(raw, os.path.join(kind, raw_file_bids), scans_fname,
+                   verbose)
```

Comment:
this is incorrect, please revert

Reply:
How so?
By running the tests slightly modified so it keeps the output and looking at the produced scans file
For example, the fif test:
```
filename	acq_time
meg\sub-01_ses-01_task-testing_run-01_meg.fif	n/a
```
and the kit file:
```
filename	acq_time
meg\sub-01_ses-01_task-testing_run-01_meg\sub-01_ses-01_task-testing_run-01_meg.sqd	2012-06-01T04:07:13
```
Both of which are the correct paths for the files.

Reply:
sorry my bad. Thanks for the fix